### PR TITLE
2017 03 07 learning to tune

### PR DESCRIPTION
**Pull request by pcunninghamatwombatsecurity**

### CODE REVIEW

File: _posts/2017-03-07-learning-to-tune.md [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+
+Describing the quality of guitar sound is as subjective as describing a bottle of wine. But even after following a well-reasoned tuning procedure...
+
+- The same note played on different strings might not share the same tuning.
+- One chord may sound in-tune, but another sounds out-of-tune.
+- Notes fretted near the headstock are more in-tune than notes fretted near the neck.
+
+Poor intonation is a mechanical deficiency in the guitar itself, and is usually caused by inferior quality components, broken tuners, maladjusted bridges, and warped necks. While it’s advisable to have your guitar “setup” by a qualified technician, or simply buy a better quality instrument, adjusting your tuning procedure can also have a dramatic effect on the sound of a guitar.
```
Comment (laurawhalin) [inline _posts/2017-03-07-learning-to-tune.md]: Ok to use "set up" instead? It's the verb form.

File: _posts/2017-03-07-learning-to-tune.md [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+
+Within a year or two, this became the "tuning ritual" that I used for many, many years.
+
+1. If the bass is not in tune; tune the open‘A’ string to a 440hz ‘A’ reference pitch and skip to step 3.
```
Comment (laurawhalin) [inline _posts/2017-03-07-learning-to-tune.md]: missing space before first 'A'

File: _posts/2017-03-07-learning-to-tune.md [hunk 3]
```diff
@@ -0,0 +1,129 @@
...
+2. Tune the open ‘A’ string an octave above the bass player’s ‘A’ string.
+3. Play the 5th fret on the ‘E’ string and tune it to match the open ‘A’ string.
+4. Play a 5th chord with the open ‘E’ and 2nd fret ‘A’ strings and tweak the tuning of the ‘E’ string until it sounds better.
+5. Play the 5th fret of the ‘A’ string, and tune the open ‘D’ string to match.
+6. Play a 5th chord with the open ‘A’ and 2nd fret ‘D’ strings and tweak the tuning of the ‘D’ string until it sounds better.
+7. Play a “power chord” using the open “E” string and 2nd fret ‘A’ and ‘D’ strings. If this doesn’t sound good return to step 3.
```
Comment (laurawhalin) [inline _posts/2017-03-07-learning-to-tune.md]: put 'E' in single quotes like the other notes.

File: _posts/2017-03-07-learning-to-tune.md [hunk 4]
```diff
@@ -0,0 +1,129 @@
...
+8. Play the 5th fret of the ‘D’ string, and tune the open ‘G’ string to match.
+9. Play a “power chord” using the open ‘A’ string and the 2nd fret ‘D’ and ‘G’ strings. If this doesn’t sound good return to step 5.
+10. Play the 4th fret of the ‘G’ string, and tune the open ‘B’ string to match.
+11. Play a 5th chord with the open ‘G’ string and 3rd fret ‘B’ strings and tweak the tuning of the ‘B’ string until it sounds better.
+12. Play the 5th fret of the ‘B’ string, and tune the open ‘E’ string to match.
+13. Play the open ‘B’ and ‘E’ strings and tweak the tuning of the ‘E’ string until it sounds better.
+14. Play some open chords like ‘A’, ‘D’, ‘G’, ‘C’, and ‘E’ and tweak notes that sound “off”.
+15. Repeat as necessary.
+
+Honestly, this was a lot of work compared to the Seven Steps. And a number conditionals, assertions, and feedback-loops have been inserted to ensure it is completed correctly. Although this is ear-intensive, and can take a few minutes to complete, many players maintain a complicated process such as this to accommodate an old or broken instrument. Sometimes the sound they get is worth the trouble.
```
Comment (laurawhalin) [inline _posts/2017-03-07-learning-to-tune.md]: Add the word "of" to "And a number conditionals..." ?

File: _posts/2017-03-07-learning-to-tune.md [hunk 5]
```diff
@@ -0,0 +1,129 @@
...
+
+### Finale
+
+This is a software development blog, and I used this out-of-context example to illustrate how non-programmers rely on the same principles that software developers use to continually create, adapt, evolve, and revolutionize their own processes:
+
+- Processes begin as a simple sequence of repeatable steps leading to a desirable outcome.
+- After significant repetition, incremental tweaks are added to improve the original algorithm.
+- Better quality platforms yield quicker and better quality results.
+- A compromise can have long-lasting effects. Others become irrelevant over time.
+- Innovation is instantly recognizable as a break in tradition or paradign shift.
```
Comment (laurawhalin) [inline _posts/2017-03-07-learning-to-tune.md]: misspelling of "paradigm"

File: _posts/2017-03-07-learning-to-tune.md [hunk 6]
```diff
@@ -0,0 +1,129 @@
+---
+layout: post
+title: Learning To Tune
+category: development
+tags: development
+author_github:  pcunninghamatwombatsecurity
+date: 2017-03-07 06:00:00
```
Comment (laurawhalin) [inline _posts/2017-03-07-learning-to-tune.md]: update to 2017-03-08, and the filename too, if you don't mind :-)

File: _posts/2017-03-07-learning-to-tune.md (hunk 1 above)
Comment (pcunninghamatwombatsecurity) [inline _posts/2017-03-07-learning-to-tune.md]: in guitar-speak, "setup" refers to how the strings are set up (noun).

"set up" is the verb usage.

File: _posts/2017-03-07-learning-to-tune.md (hunk 2 above)
Comment (pcunninghamatwombatsecurity) [inline _posts/2017-03-07-learning-to-tune.md]: ah, mullins found that one too. :)

File: _posts/2017-03-07-learning-to-tune.md (hunk 1 above)
Comment (laurawhalin) [inline _posts/2017-03-07-learning-to-tune.md]: Everyone uses it incorrectly...it's one of my pet peeves. It's being used as a verb here, which is why I pointed it out. Ultimately, it's your call :-)

**Pull request merged**